step.yml

```diff
@@ -1,12 +1,25 @@
 title: Xcode Test for iOS
 summary: |-
-  Runs Xcode's `test` action on an iOS project.
+  Runs your project's pre-defined Xcode tests on every build.
 description: |-
-  Runs Xcode's `test` action on an iOS project.
+  This Steps runs all those Xcode tests that are included in your project. If the tests are written correctly, you do not even have to configure this Step. The Step will work if your project has test targets and your Workflow has the **Deploy to Bitrise.io** Step which exports the test results and (code coverage files if needed) to the Test Reports page. This Step does not need any code signing files since the Step deploys only the test results to [bitrise.io](https://www.bitrise.io).
+  
+  ### Configuring the Step
+  If you click into the Step, there are some required input fields whose input must be set in accordance with the Xcode configuration of the project.  The **Scheme name** input field must be marked as Shared in Xcode. The **Device**, **OS version**, **Platform** input fields must be set to the value that's shown in Xcode’s device selection dropdown menu.
+  If you wish to export code coverage files as well, set the "Generate code coverage files?" to `yes`.
 
-  Write the tests and run them on every build just to make sure those tiny
-  code goblins didn't put something in the code that shouldn't be there while
-  you were at the daily Scrum meeting.
+  ### Troubleshooting
+  If the **Deploy to Bitrise.io** Step is missing from your Workflow, then the **Xcode for iOS** Step will not be able to export the test results on the Test Reports page and you won't be able to view them either.
```

Comment:
"Xcode for iOS" should be "Xcode Test for iOS"

Reply:
also fixed the lines :)